CHANGELOG.md

```diff
@@ -1,5 +1,43 @@
 # Change Log
 
+## 6.17.0 - 2018-02-14
```

Comment:
Do not see the client-fastest-connecting-ssl-uri-cache-period description in the changelog. It should point to https://github.com/vrk-kpa/xroad-joint-development/issues/25 and partially(?) to https://github.com/vrk-kpa/xroad-joint-development/issues/58